### PR TITLE
fix: fix missing comma for manual imports

### LINKED ISSUE #130: Manual Importing Causes Syntax Error
**Module version**
1.4.0

**Describe the bug**
https://github.com/nuxt-community/vuetify-module/pull/125/files#diff-a3005012a924761312a85f25454cffe4R28 missing comma between properties

**To Reproduce**

1. Set `treeShake` to 
```
        {
            components: [
                'VSnackbar',
                'VBtn',
                'VIcon'
            ],
            directives: [
                'VScroll',
                'VTouch'
            ]
        }
```
2. Run `nuxt`
3. Got error: 
```
Syntax Error: Unexpected token, expected "," (13:4)
  11 |     components: { VSnackbar, VBtn, VIcon }
  12 | 
> 13 |     directives: { VScroll, VTouch }
     |     ^
  14 | })
  15 | 
  16 | export default (ctx) => {

```

**Expected behavior**
It should have no compiling errors

### PR DESCRIPTION
**Pull request by Yihao-G**
Fixes #130

### CODE REVIEW

File: lib/templates/plugin.js [hunk 1]
```diff
@@ -20,17 +20,11 @@ import { <%= options.treeShake[lib.key].join(', ') %> } from '<%= lib.location %
...
-    if (options.treeShake && options.treeShake[lib.key] && options.treeShake[lib.key].length > 0) {
-    %>
-    <%= lib.key %>: { <%= options.treeShake[lib.key].join(', ') %> }
-    <%
-    }
-  }
-}
-%>
+<% if (options.treeShake) { %>
+<%= libImports.filter(lib => Array.isArray(options.treeShake[lib.key]) && options.treeShake[lib.key].length > 0)
```
Comment (aldarund) [inline lib/templates/plugin.js]: if it isnt array it will just silently ignored and not working for user for unknown apparent reason for user

Comment (kevinmarrec) [reply lib/templates/plugin.js]: @aldarund Is right, you just need to check that `options.treeShake[lib.key]` exists.

**Comment by kevinmarrec** [comment]
@Yihao-G Would you mind about updating the test to make it use multiple dynamic components ? (the edge case that is throwing errors for now)

**Comment by Yihao-G** [comment]
@kevinmarrec I received an email from your comment about updating the test but I couldn't find it here. Did you later delete it?

**Comment by kevinmarrec** [comment]
@Yihao-G Yes sorry, I deleted it cause i'm currently heavily refactoring tests 😁

File: lib/templates/plugin.js [hunk 2]
```diff
@@ -20,17 +20,11 @@ import { <%= options.treeShake[lib.key].join(', ') %> } from '<%= lib.location %
...
-    %>
-    <%= lib.key %>: { <%= options.treeShake[lib.key].join(', ') %> }
-    <%
-    }
-  }
-}
-%>
+<% if (options.treeShake) { %>
+<%= libImports.filter(lib => options.treeShake[lib.key] && options.treeShake[lib.key].length > 0)
+      .map(lib => `  ${lib.key}: { ${options.treeShake[lib.key].join(', ')} }`)
```
Comment (kevinmarrec) [inline lib/templates/plugin.js]:
```suggestion
      .map(lib => `  ${lib.key}: { ${options.treeShake[lib.key].join(', ') }`)
```

Comment (kevinmarrec) [reply lib/templates/plugin.js]: Nvm lol
